src/components/drag-and-drop-list/DragAndDropList.tsx

```diff
@@ -0,0 +1,137 @@
+import {useRef, useState} from "react";
+import "./drag-and-drop-list.css";
+import {DragAndDropTodoItem} from "../drag-and-drop-todo-item/DragAndDropTodoItem";
+import {GhostItem} from "../ghost-item/GhostItem";
+
+interface DragAndDropListProps {
+  title: string
+  listItems: Array<string>
+  listId: number
+  setListItems: (listItems: Array<string>) => void;
+  onItemDropped?: (item: string, sourceListId: number, targetListId: number) => void;
+}
+interface Coordinates {
```

Comment:
No need for this interface, Typescript will implicitly handle this for you when you initialize the state. (As long as this will be a single value type, not undefined etc.)